test.py

```diff
@@ -222,8 +222,8 @@ def inference_args(parser, remaining_args):
     return inf_parser.parse_args()
 
 
-def test_main(input_files, samplerate, niter, alpha, softmask, residual_model, model,
-              targets=('vocals', 'drums', 'bass', 'other'), outdir=None, no_cuda=False):
+def test_main(input_files=None, samplerate=44100, niter=1, alpha=1.0, softmask=False, residual_model=False,
+              model='umxhq', targets=('vocals', 'drums', 'bass', 'other'), outdir=None, no_cuda=False):
```

Comment:
Please add `start_stop` from #10. Thanks

Reply:
okay

Reply:
@faroit please check now, should be good.